Resources/Prototypes/DeltaV/Roles/Jobs/Legal/clerk.yml

```diff
@@ -0,0 +1,33 @@
+- type: job
+  id: Clerk
+  name: job-name-clerk
+  description: job-description-clerk
+  playTimeTracker: JobClerk
+  antagAdvantage: 2 # DeltaV - Reduced TC: Security Radio and Access
+  requirements:
+    - !type:OverallPlaytimeRequirement
+      time: 36000 # 10 hrs
+    - !type:RoleTimeRequirement 
+      department: JobLawyer
+      time: 72000 # 20 hours
+  startingGear: ClerkGear
+  icon: "JobIconClerk"
+  requireAdminNotify: true
+  supervisors: job-supervisors-cj
+  access:
+  - Service
+  - Lawyer
+  - Maintenance
+
+- type: startingGear
+  id: ClerkGear
+  equipment:
+    jumpsuit: ClothingUniformJumpsuitClerk
+    back: ClothingBackpackFilled
+    shoes: ClothingShoesBootsLaceup
+    pocket1: RubberStampNotary
```

Comment:
Like other stamps, this should be put in the locker so it can be used in the case of no available Clerks, or for a player promotion to Clerk.

Reply:
oh yeah I meant to change that and add lockers but I forgot :0